src/components/user-upload/index.tsx

```diff
@@ -140,17 +170,37 @@ function FileUploadTabContent({ onSelect }: { onSelect: (file: string) => void }
 
   return (
     <div className="flex flex-col items-center justify-center w-full h-full">
-      <label
-        htmlFor="file-upload"
-        className="flex flex-col items-center justify-center w-64 h-32 border-2 border-dashed rounded-lg cursor-pointer hover:border-yellow-500"
+      <div
+        className={cn(
+          'flex flex-col items-center justify-center w-64 h-32 border-2 border-dashed rounded-lg cursor-pointer transition-colors',
+          isDragOver
+            ? 'border-yellow-500 bg-yellow-50 dark:bg-yellow-950/20'
+            : 'border-gray-300 hover:border-yellow-500'
+        )}
+        onDragOver={handleDragOver}
+        onDragLeave={handleDragLeave}
+        onDrop={handleDrop}
+        onClick={() => document.getElementById('file-upload')?.click()}
```

Comment:
`onDragOver` `onDragLeave` and `onDrop` doesn't exist on `<label>` hence the change to `<div>`